src/frontend/src/pages/AdminToolsPage/EditGuestView/EditLogo.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { useCurrentOrganization, useOrganizationLogo, useSetOrganizationLogo } from '../../../hooks/organizations.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import EditLogoForm, { EditLogoInput } from './EditLogoForm';
+import { useToast } from '../../../hooks/toasts.hooks';
+import { Box, Card, Typography, useTheme } from '@mui/material';
+import LogoDisplay from '../../HomePage/components/LogoDisplay';
+import { NERButton } from '../../../components/NERButton';
+
+const EditLogo = () => {
+  const { data: organization, isLoading: organizationIsLoading } = useCurrentOrganization();
+  const { data: imageData, isLoading: imageDataIsLoading } = useOrganizationLogo();
```

Comment:
add error handling for these queries. Ensure all useQueries added have proper error handling.